projects/core/src/lib/layers/symbology/raster-symbology-editor/raster-symbology-editor.component.ts

```diff
@@ -53,6 +59,18 @@ export class RasterSymbologyEditorComponent implements OnInit {
     ngOnInit(): void {
         // always work on a copy in order to being able to reset changes
         this.symbology = this.layer.symbology.clone();
+
+        const bandIndex = (this.symbology.colorizer as SingleBandRasterColorizer).band;
+
+        this.projectService.getWorkflowMetaData(this.layer.workflowId).subscribe((resultDescriptor) => {
```

Comment:
do we need to track the subscription (for destruction) ?

Reply:
0e8a6ca832ce622067b4a39d679f38c8518c76fe